dev-packages/e2e-tests/test-applications/nestjs/tests/errors.test.ts

```diff
@@ -28,3 +31,19 @@ test('Sends exception to Sentry', async ({ baseURL }) => {
     span_id: expect.any(String),
   });
 });
+
+test('Does not send expected exception to Sentry', async ({ baseURL }) => {
+  const errorEventPromise = waitForError('nestjs', event => {
+    return !event.type && event.exception?.values?.[0]?.value === 'This is an unexpected exception with id 123';
+  });
+
+  const response = await fetch(`${baseURL}/test-expected-exception/123`);
+  expect(response.status).toBe(403);
+
+  const errorEvent = await errorEventPromise;
+
+  console.log("EXPECTED EXCEPTION");
+  console.log(errorEvent);
+
```

Comment:
```suggestion
```

I guess a leftover of debugging? :D

Reply:
Yes! Sorry the test was not done yet, I just pushed for debugging. Should have put it back into draft mode :)